Tests/Acceptance/CheckoutTestCase.php

```diff
@@ -178,4 +178,18 @@ public function assertPaymentSuccessful()
     {
         $this->assertTrue($this->isThankYouPage($this->getLocation()), 'Payment was not successful.');
     }
+
+    /**
+     * Checks if the given URL is pointing to the Masterpass redirect page.
+     *
+     * @param string $sUrl
+     *
+     * @return bool
+     */
+    public function isMasterpassRedirectPage($sUrl)
```

Comment:
This is specific to the Masterpass test, so it should be in `Tests/Acceptance/MasterpassCheckoutTest.php`.

Reply:
Addition: This is not needed if there is a generic `assertRedirected` method.